packages/utils/src/cmds/transform_translate.js

```diff
@@ -0,0 +1,381 @@
+            encoding: 'utf8',
+        })
+        const lines = contents
+            .split('\n')
+            .filter(line => line !== '' && !line[0].match(/\s*#/))
+
+        if (
+            languagesToTransform.length &&
+            // Primary language needed to build other translation files
+            language !== argv.primaryLanguage &&
+            languagesToTransform.indexOf(language) === -1
+        ) {
+            return mappings
+        }
+
+        mappings[language] = lines.reduce((mapping, line) => {
+            const [key, ...textParts] = line.split('=')
+            const text = textParts.join('=')
+
+            mapping[key] = text
+            return mapping
+        }, {})
+
+        return mappings
+    }, {})
+
+    for (const language in translations) {
+        if (translations.hasOwnProperty(language)) {
+            if (
+                languagesToTransform.length &&
+                languagesToTransform.indexOf(language) === -1
+            ) {
+                continue
+            }
+
+            const newLanguageFileName =
+                language === argv.primaryLanguage
+                    ? `${language}.pot`
+                    : `${language}.po`
+
+            let newContents = ''
+            const languageTranslations = translations[language]
+            const newLanguageFilePath = path.join(
+                argv.outDir,
+                newLanguageFileName
+            )
+            const shouldAppendContents =
+                !argv.overrideExistingFiles &&
+                fs.existsSync(newLanguageFilePath) &&
+                argv.appendToExistingFiles
+
+            if (
+                !argv.overrideExistingFiles &&
+                fs.existsSync(newLanguageFilePath) &&
+                !argv.appendToExistingFiles
+            ) {
+                console.log('')
+                log.debug(
+                    `Creating translation file for "${language}" :: Skipped`
+                )
+                log.debug(
+                    `Translation file ("${newLanguageFilePath}") already exists.`
+                )
+                log.debug(
+                    `If you want to append the translations, use the "--append-to-existing-files" option.`
+                )
+                log.debug(
+                    `If you want to override the existing files, use the "--override-existing-files" option`
+                )
+
+                continue
+            } else if (!fs.existsSync(newLanguageFilePath)) {
+                newContents +=
+                    language === argv.primaryLanguage
+                        ? getTemplateMainLanguage()
+                        : getTemplateAlternativeLanguage(argv.appName, language)
+            }
+
+            for (const key in languageTranslations) {
+                if (languageTranslations.hasOwnProperty(key)) {
+                    if (!translations[argv.primaryLanguage][key]) {
+                        argv.logMissingKeys &&
```

Comment:
I don't think it should be necessary to have a specific `--log-missing-keys` option - if we use `log.debug` it only shows up when verbose anyway.  If we want to make them show up when **not** verbose, then maybe it makes sense to have an option.  Is there a reason you **wouldn't** want to know about missing keys?

Reply:
It can be a very high number of keys missing (that's what happened to me in the maintenance app), it rendered all other output useless..

I thought about adding a `--dry-run` option, which could be combined with `--log-missing-keys` and `--languages` to get a list of all missing keys without any debug output, what do you think?

Reply:
OK cool, I think either `--log-missing-keys` or `--dry-run` is fine, whichever you prefer.  We could eventually add a tool for analyzing the modern i18n and finding missing keys, but for now I don't think it's a crucial step of the i18n modernize process. 👍